apps/bfd-pipeline/bfd-pipeline-rda-grpc-apps/pom.xml

```diff
@@ -17,12 +17,12 @@
 
     <properties>
         <!-- Version of gRPC related dependencies -->
-        <grpc_java.version>1.37.0</grpc_java.version>
+        <grpc_java.version>1.47.0</grpc_java.version>
         <!-- Version of the plugin used to build the gRPC stubs from the Health.proto file. -->
         <protoc.version>3.15.8</protoc.version>
         <hibernate.version>5.4.31.Final</hibernate.version>
         <jpa.version>1.0.2.Final</jpa.version>
-        <postgresql.version>42.2.20</postgresql.version>
+        <postgresql.version>42.4.0</postgresql.version>
```

Comment:
@JeremyIR I just started a PR (#1189) that will fix this.  We'll need to take a closer look at the change in the GRPC and protobuf versions to verify that works with our client code.  If you want to remove it in this PR, I can address that change in #1189

Reply:
Cool, just took those changes out of the PR.